extra_data/run_files_map.py

```diff
@@ -94,7 +94,17 @@ def load(self):
         loaded_data = []
         t0 = time.monotonic()
 
+        paths_mtimes = []
+
         for path in self.candidate_paths:
+            try:
+                st = os.stat(path)
+                paths_mtimes.append((path, st.st_mtime))
+            except (FileNotFoundError, PermissionError):
```

Comment:
Strictly speaking these exceptions can no longer occur in the `except` below when reading the file, shall we exclude them there?

Reply:
I thought about this, but I realised that the file could be deleted between the `stat()` call and `open()`. Not very likely, as our code never deletes these files, but it's easy to keep catching the error.

PermissionError could occur even without unlucky timing, because `stat()` doesn't need read permission on the file itself, only on the directory.